.gitignore

```
@@ -26,3 +26,5 @@ tests/data/
 
 # Temporary folder to refresh SDK with TypeSpec.
 TempTypeSpecFiles/
+
+sdk/**/review/*.rust.json
```

Let's just igore the whole review directory anywhere.
```suggestion
review/
```
If there's every a directory named that, we'll have to change that or have a more localized .gitignore (you can have multiple, unlike .gitattributes).